alacritty/src/window_context.rs

```diff
@@ -415,7 +420,21 @@ impl WindowContext {
         event: WinitEvent<Event>,
     ) {
         match event {
-            WinitEvent::AboutToWait | WinitEvent::RedrawRequested(_) => {
+            WinitEvent::AboutToWait => {
+                // Skip further event handling with no staged updates.
+                if self.event_queue.is_empty() {
+                    // Request a redraw in-case we have dirty flag set without events in the queue.
+                    // This could happen when event excluded from batching set `dirty` flag, but
+                    // forgot to call `redraw_requested`.
```

Comment:
This just sounds like a workaround. Either the event which "forgot" to call `redraw_requested` should be fixed or the comment needs rewording. Right now it just sounds redundant unless other code is buggy.

Reply:
I mean, I could just remove this.